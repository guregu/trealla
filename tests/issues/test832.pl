:- initialization(main).
:- use_module(library(dcgs)).

main :-
	phrase(({fail},1),_).
