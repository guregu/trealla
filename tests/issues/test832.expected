   Warning: Initialization goal exception: error(type_error(callable,((fail,_4=_5),1)),call/1).
